### DISCUSSION TITLE
spelling correction for bug reporting template

### DISCUSSION THREAD

**Discussion by Ryder17z** [Floorp-Projects/Floorp#1184] [Feature Request]
Currently:
- [X] I have checked this problem is not **ocurring** in the latest version of Firefox ESR.

Fixed:
- [X] I have checked this problem is not occurring in the latest version of Firefox ESR.

**Comment by breadbeforerice** [comment]
Currently:

 I have tried to create a new profile and the issue still occurs.

Another sentence.
English is not my native language, so it may be correct, but I feel uncomfortable when I translate it.
This representation means that no new profile is actually created.

**Comment by CutterKnife** [comment]
https://github.com/Floorp-Projects/Floorp/pull/1202
